test/onnx/test_pytorch_onnx_no_runtime.py

```diff
@@ -1187,6 +1188,22 @@ def sym_scatter_max(g, src, index, dim, out, dim_size):
                 do_constant_folding=True,
             )
 
+    @common_utils.parametrize("fp8_dtype", [torch.float8_e4m3fn, torch.float8_e5m2])
+    def test_fp8_export(self, fp8_dtype: torch.dtype):
+        class Model(torch.nn.Module):
+            def forward(
+                self,
+                x
+            ):
+                return x.to(torch.float32)
+
+        x = torch.randn(2, 3).to(fp8_dtype)
+
+        with tempfile.NamedTemporaryFile() as model_proto_file:
+            torch.onnx.export(
+                Model(), x, model_proto_file.name, opset_version=18
```

Comment:
```suggestion
                Model(), x, model_proto_file.name, opset_version=19
```
Should it be opset 19?